src/rf/apps/home/views.py

```diff
@@ -205,3 +205,25 @@ def _get_layers(request, crit=None):
     """
     models = _get_layer_models(request, crit)
     return [m.to_json() for m in models]
+
+
+@api_view
+@accepts('POST')
+@csrf_exempt
+def update_status(request):
+    """
+    Update processing status on a Layer.
+    """
+    if request.method == 'POST':
+        layer_id = request.POST['layer_id']
```

Comment:
If we want the ability to cancel/restart processing tasks I think we'll need to make a distinction between layers and jobs. But I agree it isn't critical to have right now.